### PR TITLE
update projects bank refactoring

### PR DESCRIPTION
**Pull request by MijhailVillarroel**
Reviewed that unit test for projects bank
The class Bank is refactoring  
@carledriss 
@DanielRGonzalesV
@HenrrySalinasJala

### CODE REVIEW

**Comment by carledriss** [comment]
@MijhailVillarroel 
Don't forget to update this PR

Coordinate with Henrry and Daniel to finish this task

**Comment by MijhailVillarroel** [comment]
@carledriss  
add java doc 
and rerefactoring

File: src/main/java/Bank.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+     * @param numberString
+     * @return List the string numbers
+     */
+    public List<String> readEntry(String numberString) {
+        String line1 = numberString.substring(0, 27);
+        String line2 = numberString.substring(27, 54);
+        String line3 = numberString.substring(54, 81);
+        List<String> list = new ArrayList<>();
+        int j = 0;
+        for (int i = 3; i <= 27; i += 3) {
```
Comment (carledriss) [inline src/main/java/Bank.java:65]: magic number for 27
It is not understandable at first time.

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**